OrbitGl/GpuTrack.cpp

```diff
@@ -240,3 +300,36 @@ std::string GpuTrack::GetHwExecutionTooltip(const TimerInfo& timer_info) const {
       capture_data->GetThreadName(timer_info.thread_id()), timer_info.thread_id(),
       GetPrettyTime(TicksToDuration(timer_info.start(), timer_info.end())).c_str());
 }
+
+std::string GpuTrack::GetCommandBufferTooltip(
+    const orbit_client_protos::TimerInfo& timer_info) const {
+  return absl::StrFormat(
+      "<b>Command Buffer Execution</b><br/>"
+      "<i>At `vkBeginCommandBuffer` and `vkEndCommandBuffer` `vkCmdWriteTimestamp`s have been "
+      "inserted. The gpu timestamps get aligned with the corresponding submit's hardware "
```

Comment:
Although it seems many Vulkan users use "submit" that way, I don't think it's actually a noun. I think it should be "submission": "The GPU timestamps get aligned with the corresponding hardware execution of the submission." 

Also note that we write "GPU" with capitals in other places.

Reply:
Done.